site/_docs/reference.md

```diff
@@ -3080,6 +3080,10 @@ Note:
   * A scalar value has length 1;
   * The length of array or object is the number of elements is contains.
 
+### Babel parser extensions
```

Comment:
could you instead add the new rule to the grammar and make this a note after the grammar rule is described?